README.rst

```diff
@@ -0,0 +1,28 @@
+=============
+Graph Mapping
+=============
+
+To start this exercise, we'll look at a complete Ocean program that uses the package dwave-networkx.  D-Wave NetworkX is an extension of NetworkX—a Python language package for exploration and analysis of networks and network algorithms—for users of D-Wave Systems.  The base problem of this exercise is the antenna selection problem from the `D-Wave Collection of Examples <https://github.com/dwave-examples/antenna-selection>`_.
```

Comment:
Perhaps also break the lines in this README.rst at column 80 (i.e. have each line of text have at most 79 characters). While most text editors should have text wraparound, this isn't always the case.

pep8: https://www.python.org/dev/peps/pep-0008/#maximum-line-length